nextstrain_profiles/nextstrain-open/builds.yaml

```diff
@@ -85,20 +85,13 @@ conda_environment: "workflow/envs/nextstrain_with_nextalign.yaml"
 # will start by downloading that file and proceeding straight to
 # subsampling
 inputs:
-  - name: genbank
-    metadata: "s3://nextstrain-data/ncov-ingest/genbank_metadata.tsv.gz"
-    sequences: "s3://nextstrain-data/ncov-ingest/genbank_sequences.fasta.gz"
-    # ---------------------- NOTE --------------------------
-    # PR 550 changed the path of intermediate files on
-    # the S3 bucket to include the origin (e.g. _gisaid).
-    # Some of these files may not yet exist!
-    # ------------------------------------------------------
-    # metadata: "s3://nextstrain-ncov-private/metadata_gisaid.tsv.gz"
-    # sequences: "s3://nextstrain-ncov-private/sequences_gisaid.fasta.gz"
-    # aligned: "s3://nextstrain-ncov-private/aligned_gisaid.fasta.xz"
-    # to-exclude: "s3://nextstrain-ncov-private/to-exclude_gisaid.txt.xz"
-    # masked: "s3://nextstrain-ncov-private/masked_gisaid.fasta.fasta.xz"
-    # filtered: "s3://nextstrain-ncov-private/filtered_gisaid.fasta.xz"
+  - name: open
+    metadata: "s3://nextstrain-data/files/ncov/open/metadata.tsv.gz"
+    sequences: "s3://nextstrain-data/files/ncov/open/sequences.fasta.gz"
+    # aligned: "s3://nextstrain-files/files/ncov/open/aligned.fasta.xz"
```

Comment:
Is this a typo or do we actually have a `nextstrain-files` bucket?

Either way, won't only the metadata & sequences files be available, or are we maintaining an all-open-data alignment etc? My understanding is that this profile will only be able to start from `metadata` and `sequences`, but I may be out-of-date here!